src/main/handlers/exportProjectHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { BrowserWindow, dialog } from 'electron';
+import { IpcContext } from 'main/types';
+import { Project } from '../../sharedTypes';
+import { exportEDL } from '../export';
+
+const getExportFilePath: (
+  mainWindow: BrowserWindow | null,
+  project: Project
+) => Promise<string> = async (mainWindow, project) => {
+  if (mainWindow === null) {
+    throw new Error('Main window not defined');
+  }
+
+  const dialogResponse = await dialog.showSaveDialog(mainWindow, {
+    defaultPath: project.name,
+    filters: [{ name: '.edl Files', extensions: ['edl'] }],
+    buttonLabel: 'Export',
+    title: 'Export Project',
+    properties: ['createDirectory'],
+  });
+
+  if (dialogResponse.canceled) {
+    throw new Error('Dialog cancelled');
+  }
+
+  return dialogResponse.filePath as string;
+};
+
+const exportProjectToFile: (
```

Comment:
I think the type syntax for the functions in this file should be the same as for the other handlers. There was a PR to make them all consistent in preparation for the IPC handler gen code PR.

Reply:
done